Source/WebCore/rendering/ReferencedSVGResources.h

```diff
@@ -59,6 +60,8 @@ class ReferencedSVGResources {
     RenderSVGResourceClipper* referencedClipperRenderer(TreeScope&, const ReferencePathOperation&);
     SVGFilterElement* referencedFilterElement(TreeScope&, const ReferenceFilterOperation&);
 
+    RenderSVGResourceContainer* referencedRenderResource(TreeScope&, const AtomString& fragment);
```

Comment:
I think this could be `static` or at lest `const` ?

Reply:
Fixed. It is made static.